en_ID/pages/faq.md

```diff
@@ -210,6 +210,72 @@ Tip: If clip isn't working, you can locate the hidden .ssh folder, open the file
 
 * If prompted, confirm your GitHub password.
 
+## Add new SSH key for Linux
+
+To configure your GitHub account to use your new (or existing) SSH key, you'll also need to add it to your GitHub account.
+
+### Generating a new SSH key
+
+1. Open Terminal.
+2. Paste the text below, substituting in your GitHub email address.
+
+```
+$ ssh-keygen -t ed25519 -C "your_email@example.com"
+
+Note: If you are using a legacy system that doesn't support the Ed25519 algorithm, use:
+
+$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+```
+
+This creates a new ssh key, using the provided email as a label.
+
+```
+> Generating public/private ed25519 key pair.
+```
+
+3. When you're prompted to "Enter a file in which to save the key," press Enter. This accepts the default file location.
+
+```
+> Enter a file in which to save the key (/home/you/.ssh/id_ed25519): [Press enter]
+```
+
+4. At the prompt, type a secure passphrase.
+
+```
+> Enter passphrase (empty for no passphrase): [Type a passphrase]
+> Enter same passphrase again: [Type passphrase again]
+```
+
+### Adding your SSH key to the ssh-agent
+1. Start the ssh-agent in the background.
+```
+$ eval "$(ssh-agent -s)"
+> Agent pid 59566
+```
+2. Add your SSH private key to the ssh-agent. If you created your key with a different name, or if you are adding an existing key that has a different name, replace id_ed25519 in the command with the name of your private key file.
+```
+$ ssh-add ~/.ssh/id_ed25519
+```
+
+### Add the SSH key to your GitHub account.
+1. Copy the SSH public key to your clipboard. If your SSH public key file has a different name than the example code, modify the filename to match your current setup. When copying your key, don't add any newlines or whitespace.
+```
+$ sudo apt-get update
+$ sudo apt-get install xclip
+# Downloads and installs xclip. If you don't have `apt-get`, you might need to use another installer (like `yum`)
+$ xclip -selection clipboard < ~/.ssh/id_ed25519.pub
+#Copies the contents of the id_ed25519.pub file to your clipboard
```

Comment:
This is nitpicking, but you can add more space after the hashtag.

Reply:
oh oke.. i have revisied it, please review ya..

Reply:
oh oke.. i have revisied it, please review ya..